.github/workflows/ci.yml

```diff
@@ -19,9 +36,14 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: install, build, and test
       run: |
+        npm install -g npm
         npm install
+        npm install --no-save @fortawesome/fontawesome-svg-core@${{ matrix.fontawesome-svg-core }} @fortawesome/free-solid-svg-icons@${{ matrix.free-solid-svg-icons }} vue@${{ matrix.vue }}
         npm run build
-        npm run test
+        npm install --no-save vue@${{ matrix.vue }}
```

Comment:
@jasonlundien are we installing `vue` twice?

Reply:
yes we are, so I will remove this line !